csfieldguide/chapters/content/en/human-computer-interaction/sections/user-experience.md

```diff
@@ -0,0 +1,559 @@
+
+{panel end}
+
+### Step 4: Describe your user.
+
+**UX methods: Empathy map, user personas and user stories**
+
+Now that we’ve established who our users are and what they want, it would be useful to create *user personas*.
+A *user persona* is a hypothetical character that is representative of your target user.
+To help us with creating a user persona, we will use an *empathy map* to give us a structure to work with.
+An *empathy map* defines four aspects of the persona in quadrants: how they Think and Feel, and what they Do and Say.
+We’ve created a very basic one below to help us develop a persona for the target users of our sleep habits app:
+
+|                                                                                                                                          	| **User**                                                                                                                                                                                                                    	|
+|------------------------------------------------------------------------------------------------------------------------------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| **Says**<br>Things the user says out loud in an interview or wrote in their survey.<br><br>*I am so tired.<br>I think I stayed up too late last night.* 	| **Thinks**<br>What do they think about? What are their needs, goals and motivations?<br><br>*I wish I could get more sleep.<br>Maybe I should go to bed earlier.<br>How many hours of sleep do I actually need?<br>Why am I always tired?* 	|
+| **Does**<br>What actions does the user do?<br><br>*Always hits “snooze” on the alarm.<br>Drinks coffee first thing in the morning.*                     	| **Feels**<br>Consider the tone they use in an interview and/or survey answers.<br><br>*Tired.<br>Annoyed.<br>Frustrated.*                                                                                                                	|
+
+Using our empathy map we are able to derive the target users' needs and from here we created persona representatives of our target user.
+Meet some students representing our target audience, though for this example we'll just focus on Bevan:
+
+{panel type="curiosity"}
+
+# Be aware of variety in your audience
+
+The three examples below are possible users for this application.
+When performing your own user studies, you should list and research their gender, age, culture, etc.
+
+For example: Elderly people across different cultures behave differently.
+If your target users include the elderly, you should research how their cultures might impact how they interact with your product.
```

Comment:
This section reads a bit odd since it says "Elderly people across different cultures behave differently" but in reality it's "people across different cultures behave differently". So maybe a better example would be something like:

> For example: Elderly people generally have poorer levels of fine motor skills compared to young adult. 
> If your target users include the elderly, you should research how this might impact how they interact with your product.